community-plugins.json

```diff
@@ -5707,7 +5707,7 @@
     {
         "id": "contextual-comments",
         "name": "Contextual Comments",
-        "description": "General comments or langage related (in code blocks) + 2 commands trim end lines (all doc/code blocks).",
+        "description": "General comments or language related (js, python, c++, dataviewjs, lua...) Fast and 100% efficient.",
```

Comment:
The capitalization of the language names is still wrong.

Reply:
I don't understand I put the capitalization as you can find it in codeblocks. what do you mean?

Reply:
is it ok now?

Reply:
Ok I will do some more general descriptions

Reply:
@joethei I think it's much better now. I made my descriptions more general.